aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -97,6 +97,31 @@ def validate_calc_job(inputs: Any, ctx: PortNamespace) -> Optional[str]:  # pyli
     return None
 
 
+def validate_stash_options(stash_options: Any, _: Any) -> Optional[str]:
+    """Validate the ``stash`` options."""
+    from aiida.common.datastructures import StashMode
+
+    # Normally we would check if `stash_options` is not `None, but due to a bug in `plumpy`, even if not specified, the
```

Comment:
yeah, it was fixed in `0.17.0`. I kept it in because there was a reason I couldn't rely on this version of `plumpy` being installed, but I realize now that that is for `aiida-quantumespresso`. For the current state of `aiida-core` we require `plumpy~=0.18.4` so we can indeed remove this guard.